code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -605,7 +604,7 @@
 		return TRUE
 	return FALSE
 
-/obj/vehicle/sealed/mecha/vehicle_move(direction, forcerotate = FALSE)
```

Comment:
This proc has no code paths that players can execute. It's only called in `/mob/living/simple_animal/hostile/syndicate/mecha_pilot/proc/mecha_face_target(atom/A)`

Similarly, mechs don't seem to have any riding component attached to them, rendering them completely immobile and unusable. (Witnessed with both Ripley and Phazon on live servers)